clients/cobol-lsp-vscode-extension/src/test/suite/lsp.spec.test.ts

```diff
@@ -89,7 +89,8 @@ suite("Integration Test Suite", function () {
     ?.timeout(helper.TEST_TIMEOUT)
     ?.slow(1000);
 
-  test("TC312735 Check EXEC CICS is in Procedure Division", async () => {
+  // Skip in favore of TestExecCicsInProcedureDivision java unit test
+  test.skip("TC312735 Check EXEC CICS is in Procedure Division", async () => {
```

Comment:
Is the test no longer valid?

Reply:
It is valid, do you think we should keep it?

Reply:
I think so.

Reply:
ok